pkg/kn/commands/service_update_test.go

```diff
@@ -107,3 +107,153 @@ func TestServiceUpdateImage(t *testing.T) {
 		t.Fatalf("wrong image set: %v", conf.RevisionTemplate.Spec.Container.Image)
 	}
 }
+
+func TestServiceUpdateRequestsLimitsCPU(t *testing.T) {
+	service := createMockServiceWithResources(t, "250", "64Mi", "1000m", "1024Mi")
+
+	action, updated, _, err := fakeServiceUpdate(service, []string{
+		"service", "update", "foo", "--requests-cpu", "500m", "--limits-cpu", "1000m"})
```

Comment:
did you want to have it be different than `1000m`?

Reply:
Well, wanted to verify case where the update does nothing...